lib/index.d.ts

```diff
@@ -78,6 +78,7 @@ class XlsxTemplate
     public splitRange(range : string) : RangeSplit;
     public joinRange(range : RangeSplit) : string;
     public extractPlaceholders(templateString : string) : TemplatePlaceholder[];
+    public handleImageError(substitution : any, error : Error) : Buffer;
```

Comment:
I thought that this would be part of `options`. That way developer does not need monkey-patching of the class.